Description: Upstream changes introduced in version 0.17.1.1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 yad (0.17.1.1) UNRELEASED; urgency=low
 .
   * Initial release.
 .
 The person named in the Author field signed this changelog entry.
Author: Anthony Nordquist <salinelinux@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ yad-0.17.1.1/config.log
@@ -0,0 +1,443 @@
+This file contains any messages produced by compilers while
+running configure, to aid debugging if configure makes a mistake.
+
+It was created by YAD configure 0.17.1.1, which was
+generated by GNU Autoconf 2.68.  Invocation command line was
+
+  $ ./configure --build=i486-linux-gnu --prefix=/usr --includedir=${prefix}/include --mandir=${prefix}/share/man --infodir=${prefix}/share/info --sysconfdir=/etc --localstatedir=/var --libexecdir=${prefix}/lib/yad --disable-maintainer-mode --disable-dependency-tracking
+
+## --------- ##
+## Platform. ##
+## --------- ##
+
+hostname = user
+uname -m = i686
+uname -r = 2.6.32-5-686
+uname -s = Linux
+uname -v = #1 SMP Mon Jan 16 16:04:25 UTC 2012
+
+/usr/bin/uname -p = unknown
+/bin/uname -X     = unknown
+
+/bin/arch              = unknown
+/usr/bin/arch -k       = unknown
+/usr/convex/getsysinfo = unknown
+/usr/bin/hostinfo      = unknown
+/bin/machine           = unknown
+/usr/bin/oslevel       = unknown
+/bin/universe          = unknown
+
+PATH: /usr/sbin
+PATH: /usr/bin
+PATH: /sbin
+PATH: /bin
+PATH: /usr/bin/X11
+
+
+## ----------- ##
+## Core tests. ##
+## ----------- ##
+
+configure:2256: checking for a BSD-compatible install
+configure:2324: result: /usr/bin/install -c
+configure:2335: checking whether build environment is sane
+configure:2385: result: yes
+configure:2526: checking for a thread-safe mkdir -p
+configure:2565: result: /bin/mkdir -p
+configure:2578: checking for gawk
+configure:2608: result: no
+configure:2578: checking for mawk
+configure:2594: found /usr/bin/mawk
+configure:2605: result: mawk
+configure:2616: checking whether make sets $(MAKE)
+configure:2638: result: yes
+configure:2729: checking whether make supports nested variables
+configure:2746: result: yes
+configure:2765: checking whether to enable maintainer-specific portions of Makefiles
+configure:2774: result: no
+configure:2789: checking whether NLS is requested
+configure:2798: result: yes
+configure:2814: checking for style of include used by make
+configure:2842: result: GNU
+configure:2913: checking for gcc
+configure:2929: found /usr/bin/gcc
+configure:2940: result: gcc
+configure:3169: checking for C compiler version
+configure:3178: gcc --version >&5
+gcc (Debian 4.4.5-8) 4.4.5
+Copyright (C) 2010 Free Software Foundation, Inc.
+This is free software; see the source for copying conditions.  There is NO
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+configure:3189: $? = 0
+configure:3178: gcc -v >&5
+Using built-in specs.
+Target: i486-linux-gnu
+Configured with: ../src/configure -v --with-pkgversion='Debian 4.4.5-8' --with-bugurl=file:///usr/share/doc/gcc-4.4/README.Bugs --enable-languages=c,c++,fortran,objc,obj-c++ --prefix=/usr --program-suffix=-4.4 --enable-shared --enable-multiarch --enable-linker-build-id --with-system-zlib --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.4 --libdir=/usr/lib --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-objc-gc --enable-targets=all --with-arch-32=i586 --with-tune=generic --enable-checking=release --build=i486-linux-gnu --host=i486-linux-gnu --target=i486-linux-gnu
+Thread model: posix
+gcc version 4.4.5 (Debian 4.4.5-8) 
+configure:3189: $? = 0
+configure:3178: gcc -V >&5
+gcc: '-V' option must have argument
+configure:3189: $? = 1
+configure:3178: gcc -qversion >&5
+gcc: unrecognized option '-qversion'
+gcc: no input files
+configure:3189: $? = 1
+configure:3209: checking whether the C compiler works
+configure:3231: gcc -g -O2   conftest.c  >&5
+configure:3235: $? = 0
+configure:3283: result: yes
+configure:3286: checking for C compiler default output file name
+configure:3288: result: a.out
+configure:3294: checking for suffix of executables
+configure:3301: gcc -o conftest -g -O2   conftest.c  >&5
+configure:3305: $? = 0
+configure:3327: result: 
+configure:3349: checking whether we are cross compiling
+configure:3357: gcc -o conftest -g -O2   conftest.c  >&5
+configure:3361: $? = 0
+configure:3368: ./conftest
+configure:3372: $? = 0
+configure:3387: result: no
+configure:3392: checking for suffix of object files
+configure:3414: gcc -c -g -O2  conftest.c >&5
+configure:3418: $? = 0
+configure:3439: result: o
+configure:3443: checking whether we are using the GNU C compiler
+configure:3462: gcc -c -g -O2  conftest.c >&5
+configure:3462: $? = 0
+configure:3471: result: yes
+configure:3480: checking whether gcc accepts -g
+configure:3500: gcc -c -g  conftest.c >&5
+configure:3500: $? = 0
+configure:3541: result: yes
+configure:3558: checking for gcc option to accept ISO C89
+configure:3622: gcc  -c -g -O2  conftest.c >&5
+configure:3622: $? = 0
+configure:3635: result: none needed
+configure:3657: checking dependency style of gcc
+configure:3768: result: none
+configure:3795: checking for intltool >= 0.40.0
+configure:3802: result: 0.41.1 found
+configure:3810: checking for intltool-update
+configure:3828: found /usr/bin/intltool-update
+configure:3840: result: /usr/bin/intltool-update
+configure:3850: checking for intltool-merge
+configure:3868: found /usr/bin/intltool-merge
+configure:3880: result: /usr/bin/intltool-merge
+configure:3890: checking for intltool-extract
+configure:3908: found /usr/bin/intltool-extract
+configure:3920: result: /usr/bin/intltool-extract
+configure:4051: checking for xgettext
+configure:4069: found /usr/bin/xgettext
+configure:4081: result: /usr/bin/xgettext
+configure:4091: checking for msgmerge
+configure:4109: found /usr/bin/msgmerge
+configure:4121: result: /usr/bin/msgmerge
+configure:4131: checking for msgfmt
+configure:4149: found /usr/bin/msgfmt
+configure:4161: result: /usr/bin/msgfmt
+configure:4171: checking for gmsgfmt
+configure:4202: result: /usr/bin/msgfmt
+configure:4222: checking for perl
+configure:4240: found /usr/bin/perl
+configure:4252: result: /usr/bin/perl
+configure:4263: checking for perl >= 5.8.1
+configure:4270: result: 5.10.1
+configure:4274: checking for XML::Parser
+configure:4277: result: ok
+configure:4302: gcc -o conftest -g -O2   conftest.c  >&5
+configure:4302: $? = 0
+configure:4377: checking for gcc
+configure:4404: result: gcc
+configure:4633: checking for C compiler version
+configure:4642: gcc --version >&5
+gcc (Debian 4.4.5-8) 4.4.5
+Copyright (C) 2010 Free Software Foundation, Inc.
+This is free software; see the source for copying conditions.  There is NO
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+configure:4653: $? = 0
+configure:4642: gcc -v >&5
+Using built-in specs.
+Target: i486-linux-gnu
+Configured with: ../src/configure -v --with-pkgversion='Debian 4.4.5-8' --with-bugurl=file:///usr/share/doc/gcc-4.4/README.Bugs --enable-languages=c,c++,fortran,objc,obj-c++ --prefix=/usr --program-suffix=-4.4 --enable-shared --enable-multiarch --enable-linker-build-id --with-system-zlib --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.4 --libdir=/usr/lib --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-objc-gc --enable-targets=all --with-arch-32=i586 --with-tune=generic --enable-checking=release --build=i486-linux-gnu --host=i486-linux-gnu --target=i486-linux-gnu
+Thread model: posix
+gcc version 4.4.5 (Debian 4.4.5-8) 
+configure:4653: $? = 0
+configure:4642: gcc -V >&5
+gcc: '-V' option must have argument
+configure:4653: $? = 1
+configure:4642: gcc -qversion >&5
+gcc: unrecognized option '-qversion'
+gcc: no input files
+configure:4653: $? = 1
+configure:4657: checking whether we are using the GNU C compiler
+configure:4685: result: yes
+configure:4694: checking whether gcc accepts -g
+configure:4755: result: yes
+configure:4772: checking for gcc option to accept ISO C89
+configure:4849: result: none needed
+configure:4871: checking dependency style of gcc
+configure:4982: result: none
+configure:5015: result: gtk2
+configure:5083: checking for pkg-config
+configure:5101: found /usr/bin/pkg-config
+configure:5113: result: /usr/bin/pkg-config
+configure:5138: checking pkg-config is at least version 0.9.0
+configure:5141: result: yes
+configure:5151: checking for YAD
+configure:5158: $PKG_CONFIG --exists --print-errors "$GTK_MODULE >= $GTK_REQUIRED $GTK_PRINT"
+Package gtk+-2.0 was not found in the pkg-config search path.
+Perhaps you should add the directory containing `gtk+-2.0.pc'
+to the PKG_CONFIG_PATH environment variable
+No package 'gtk+-2.0' found
+Package gtk+-unix-print-2.0 was not found in the pkg-config search path.
+Perhaps you should add the directory containing `gtk+-unix-print-2.0.pc'
+to the PKG_CONFIG_PATH environment variable
+No package 'gtk+-unix-print-2.0' found
+configure:5161: $? = 1
+configure:5174: $PKG_CONFIG --exists --print-errors "$GTK_MODULE >= $GTK_REQUIRED $GTK_PRINT"
+Package gtk+-2.0 was not found in the pkg-config search path.
+Perhaps you should add the directory containing `gtk+-2.0.pc'
+to the PKG_CONFIG_PATH environment variable
+No package 'gtk+-2.0' found
+Package gtk+-unix-print-2.0 was not found in the pkg-config search path.
+Perhaps you should add the directory containing `gtk+-unix-print-2.0.pc'
+to the PKG_CONFIG_PATH environment variable
+No package 'gtk+-unix-print-2.0' found
+configure:5177: $? = 1
+configure:5190: result: no
+No package 'gtk+-2.0' found
+No package 'gtk+-unix-print-2.0' found
+configure:5206: error: Package requirements (gtk+-2.0 >= 2.16.0 gtk+-unix-print-2.0) were not met:
+
+No package 'gtk+-2.0' found
+No package 'gtk+-unix-print-2.0' found
+
+Consider adjusting the PKG_CONFIG_PATH environment variable if you
+installed software in a non-standard prefix.
+
+Alternatively, you may set the environment variables YAD_CFLAGS
+and YAD_LIBS to avoid the need to call pkg-config.
+See the pkg-config man page for more details.
+
+## ---------------- ##
+## Cache variables. ##
+## ---------------- ##
+
+ac_cv_c_compiler_gnu=yes
+ac_cv_env_CC_set=
+ac_cv_env_CC_value=
+ac_cv_env_CFLAGS_set=set
+ac_cv_env_CFLAGS_value='-g -O2'
+ac_cv_env_CPPFLAGS_set=set
+ac_cv_env_CPPFLAGS_value=
+ac_cv_env_CPP_set=
+ac_cv_env_CPP_value=
+ac_cv_env_LDFLAGS_set=set
+ac_cv_env_LDFLAGS_value=
+ac_cv_env_LIBS_set=
+ac_cv_env_LIBS_value=
+ac_cv_env_PKG_CONFIG_LIBDIR_set=
+ac_cv_env_PKG_CONFIG_LIBDIR_value=
+ac_cv_env_PKG_CONFIG_PATH_set=
+ac_cv_env_PKG_CONFIG_PATH_value=
+ac_cv_env_PKG_CONFIG_set=
+ac_cv_env_PKG_CONFIG_value=
+ac_cv_env_YAD_CFLAGS_set=
+ac_cv_env_YAD_CFLAGS_value=
+ac_cv_env_YAD_LIBS_set=
+ac_cv_env_YAD_LIBS_value=
+ac_cv_env_build_alias_set=set
+ac_cv_env_build_alias_value=i486-linux-gnu
+ac_cv_env_host_alias_set=
+ac_cv_env_host_alias_value=
+ac_cv_env_target_alias_set=
+ac_cv_env_target_alias_value=
+ac_cv_objext=o
+ac_cv_path_GMSGFMT=/usr/bin/msgfmt
+ac_cv_path_INTLTOOL_EXTRACT=/usr/bin/intltool-extract
+ac_cv_path_INTLTOOL_MERGE=/usr/bin/intltool-merge
+ac_cv_path_INTLTOOL_PERL=/usr/bin/perl
+ac_cv_path_INTLTOOL_UPDATE=/usr/bin/intltool-update
+ac_cv_path_MSGFMT=/usr/bin/msgfmt
+ac_cv_path_MSGMERGE=/usr/bin/msgmerge
+ac_cv_path_XGETTEXT=/usr/bin/xgettext
+ac_cv_path_ac_pt_PKG_CONFIG=/usr/bin/pkg-config
+ac_cv_path_install='/usr/bin/install -c'
+ac_cv_path_mkdir=/bin/mkdir
+ac_cv_prog_AWK=mawk
+ac_cv_prog_ac_ct_CC=gcc
+ac_cv_prog_cc_c89=
+ac_cv_prog_cc_g=yes
+ac_cv_prog_make_make_set=yes
+am_cv_CC_dependencies_compiler_type=none
+am_cv_make_support_nested_variables=yes
+
+## ----------------- ##
+## Output variables. ##
+## ----------------- ##
+
+ACLOCAL='${SHELL} /home/user/SalineOS/Yad/yad-0.17.1.1/missing --run aclocal-1.11'
+ALL_LINGUAS=''
+AMDEPBACKSLASH=''
+AMDEP_FALSE=''
+AMDEP_TRUE='#'
+AMTAR='$${TAR-tar}'
+AM_BACKSLASH='\'
+AM_CFLAGS=''
+AM_CPPFLAGS=''
+AM_DEFAULT_V='$(AM_DEFAULT_VERBOSITY)'
+AM_DEFAULT_VERBOSITY='0'
+AM_LDFLAGS=''
+AM_V='$(V)'
+AUTOCONF='${SHELL} /home/user/SalineOS/Yad/yad-0.17.1.1/missing --run autoconf'
+AUTOHEADER='${SHELL} /home/user/SalineOS/Yad/yad-0.17.1.1/missing --run autoheader'
+AUTOMAKE='${SHELL} /home/user/SalineOS/Yad/yad-0.17.1.1/missing --run automake-1.11'
+AWK='mawk'
+BUILD_IB_FALSE=''
+BUILD_IB_TRUE=''
+CATALOGS=''
+CATOBJEXT=''
+CC='gcc'
+CCDEPMODE='depmode=none'
+CFLAGS='-g -O2'
+CPP=''
+CPPFLAGS=''
+CYGPATH_W='echo'
+DATADIRNAME='share'
+DEFS=''
+DEPDIR='.deps'
+DISABLE_DEPRECATED=''
+ECHO_C=''
+ECHO_N='-n'
+ECHO_T=''
+EGREP=''
+EXEEXT=''
+GETTEXT_PACKAGE=''
+GMOFILES=''
+GMSGFMT='/usr/bin/msgfmt'
+GREP=''
+INSTALL_DATA='${INSTALL} -m 644'
+INSTALL_PROGRAM='${INSTALL}'
+INSTALL_SCRIPT='${INSTALL}'
+INSTALL_STRIP_PROGRAM='$(install_sh) -c -s'
+INSTOBJEXT=''
+INTLLIBS=''
+INTLTOOL_CAVES_RULE='%.caves:     %.caves.in     $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; LC_ALL=C $(INTLTOOL_MERGE) -d -u -c $(top_builddir)/po/.intltool-merge-cache $(top_srcdir)/po $< $@'
+INTLTOOL_DESKTOP_RULE='%.desktop:   %.desktop.in   $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; LC_ALL=C $(INTLTOOL_MERGE) -d -u -c $(top_builddir)/po/.intltool-merge-cache $(top_srcdir)/po $< $@'
+INTLTOOL_DIRECTORY_RULE='%.directory: %.directory.in $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; LC_ALL=C $(INTLTOOL_MERGE) -d -u -c $(top_builddir)/po/.intltool-merge-cache $(top_srcdir)/po $< $@'
+INTLTOOL_EXTRACT='/usr/bin/intltool-extract'
+INTLTOOL_KBD_RULE='%.kbd:       %.kbd.in       $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; LC_ALL=C $(INTLTOOL_MERGE) -x -u -m -c $(top_builddir)/po/.intltool-merge-cache $(top_srcdir)/po $< $@'
+INTLTOOL_KEYS_RULE='%.keys:      %.keys.in      $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; LC_ALL=C $(INTLTOOL_MERGE) -k -u -c $(top_builddir)/po/.intltool-merge-cache $(top_srcdir)/po $< $@'
+INTLTOOL_MERGE='/usr/bin/intltool-merge'
+INTLTOOL_OAF_RULE='%.oaf:       %.oaf.in       $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; LC_ALL=C $(INTLTOOL_MERGE) -o -p $(top_srcdir)/po $< $@'
+INTLTOOL_PERL='/usr/bin/perl'
+INTLTOOL_POLICY_RULE='%.policy:    %.policy.in    $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; LC_ALL=C $(INTLTOOL_MERGE) -x -u -c $(top_builddir)/po/.intltool-merge-cache $(top_srcdir)/po $< $@'
+INTLTOOL_PONG_RULE='%.pong:      %.pong.in      $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; LC_ALL=C $(INTLTOOL_MERGE) -x -u -c $(top_builddir)/po/.intltool-merge-cache $(top_srcdir)/po $< $@'
+INTLTOOL_PROP_RULE='%.prop:      %.prop.in      $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; LC_ALL=C $(INTLTOOL_MERGE) -d -u -c $(top_builddir)/po/.intltool-merge-cache $(top_srcdir)/po $< $@'
+INTLTOOL_SCHEMAS_RULE='%.schemas:   %.schemas.in   $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; LC_ALL=C $(INTLTOOL_MERGE) -s -u -c $(top_builddir)/po/.intltool-merge-cache $(top_srcdir)/po $< $@'
+INTLTOOL_SERVER_RULE='%.server:    %.server.in    $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; LC_ALL=C $(INTLTOOL_MERGE) -o -u -c $(top_builddir)/po/.intltool-merge-cache $(top_srcdir)/po $< $@'
+INTLTOOL_SERVICE_RULE='%.service: %.service.in   $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; LC_ALL=C $(INTLTOOL_MERGE) -d -u -c $(top_builddir)/po/.intltool-merge-cache $(top_srcdir)/po $< $@'
+INTLTOOL_SHEET_RULE='%.sheet:     %.sheet.in     $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; LC_ALL=C $(INTLTOOL_MERGE) -x -u -c $(top_builddir)/po/.intltool-merge-cache $(top_srcdir)/po $< $@'
+INTLTOOL_SOUNDLIST_RULE='%.soundlist: %.soundlist.in $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; LC_ALL=C $(INTLTOOL_MERGE) -d -u -c $(top_builddir)/po/.intltool-merge-cache $(top_srcdir)/po $< $@'
+INTLTOOL_THEME_RULE='%.theme:     %.theme.in     $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; LC_ALL=C $(INTLTOOL_MERGE) -d -u -c $(top_builddir)/po/.intltool-merge-cache $(top_srcdir)/po $< $@'
+INTLTOOL_UI_RULE='%.ui:        %.ui.in        $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; LC_ALL=C $(INTLTOOL_MERGE) -x -u -c $(top_builddir)/po/.intltool-merge-cache $(top_srcdir)/po $< $@'
+INTLTOOL_UPDATE='/usr/bin/intltool-update'
+INTLTOOL_XAM_RULE='%.xam:       %.xml.in       $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; LC_ALL=C $(INTLTOOL_MERGE) -x -u -c $(top_builddir)/po/.intltool-merge-cache $(top_srcdir)/po $< $@'
+INTLTOOL_XML_NOMERGE_RULE='%.xml:       %.xml.in       $(INTLTOOL_MERGE) ; LC_ALL=C $(INTLTOOL_MERGE) -x -u --no-translations $< $@'
+INTLTOOL_XML_RULE='%.xml:       %.xml.in       $(INTLTOOL_MERGE) $(wildcard $(top_srcdir)/po/*.po) ; LC_ALL=C $(INTLTOOL_MERGE) -x -u -c $(top_builddir)/po/.intltool-merge-cache $(top_srcdir)/po $< $@'
+LDFLAGS=''
+LIBOBJS=''
+LIBS=''
+LTLIBOBJS=''
+MAINT='#'
+MAINTAINER_MODE_FALSE=''
+MAINTAINER_MODE_TRUE='#'
+MAKEINFO='${SHELL} /home/user/SalineOS/Yad/yad-0.17.1.1/missing --run makeinfo'
+MKDIR_P='/bin/mkdir -p'
+MKINSTALLDIRS=''
+MSGFMT='/usr/bin/msgfmt'
+MSGFMT_OPTS=''
+MSGMERGE='/usr/bin/msgmerge'
+OBJEXT='o'
+PACKAGE='yad'
+PACKAGE_BUGREPORT='http://code.google.com/p/yad/issues/entry'
+PACKAGE_NAME='YAD'
+PACKAGE_STRING='YAD 0.17.1.1'
+PACKAGE_TARNAME='yad'
+PACKAGE_URL='http://code.google.com/p/yad'
+PACKAGE_VERSION='0.17.1.1'
+PATH_SEPARATOR=':'
+PKG_CONFIG='/usr/bin/pkg-config'
+PKG_CONFIG_LIBDIR=''
+PKG_CONFIG_PATH=''
+POFILES=''
+POSUB=''
+PO_IN_DATADIR_FALSE=''
+PO_IN_DATADIR_TRUE=''
+SET_MAKE=''
+SHELL='/bin/bash'
+STRIP=''
+USE_NLS='yes'
+VERSION='0.17.1.1'
+XGETTEXT='/usr/bin/xgettext'
+YAD_CFLAGS=''
+YAD_LIBS=''
+ac_ct_CC='gcc'
+am__EXEEXT_FALSE=''
+am__EXEEXT_TRUE=''
+am__fastdepCC_FALSE=''
+am__fastdepCC_TRUE='#'
+am__include='include'
+am__isrc=''
+am__leading_dot='.'
+am__nodep=''
+am__quote=''
+am__tar='$${TAR-tar} chof - "$$tardir"'
+am__untar='$${TAR-tar} xf -'
+bindir='${exec_prefix}/bin'
+build_alias='i486-linux-gnu'
+datadir='${datarootdir}'
+datarootdir='${prefix}/share'
+docdir='${datarootdir}/doc/${PACKAGE_TARNAME}'
+dvidir='${docdir}'
+exec_prefix='NONE'
+host_alias=''
+htmldir='${docdir}'
+includedir='${prefix}/include'
+infodir='${prefix}/share/info'
+install_sh='${SHELL} /home/user/SalineOS/Yad/yad-0.17.1.1/install-sh'
+libdir='${exec_prefix}/lib'
+libexecdir='${prefix}/lib/yad'
+localedir='${datarootdir}/locale'
+localstatedir='/var'
+mandir='${prefix}/share/man'
+mkdir_p='/bin/mkdir -p'
+oldincludedir='/usr/include'
+pdfdir='${docdir}'
+prefix='/usr'
+program_transform_name='s,x,x,'
+psdir='${docdir}'
+sbindir='${exec_prefix}/sbin'
+sharedstatedir='${prefix}/com'
+sysconfdir='/etc'
+target_alias=''
+
+## ----------- ##
+## confdefs.h. ##
+## ----------- ##
+
+/* confdefs.h */
+#define PACKAGE_NAME "YAD"
+#define PACKAGE_TARNAME "yad"
+#define PACKAGE_VERSION "0.17.1.1"
+#define PACKAGE_STRING "YAD 0.17.1.1"
+#define PACKAGE_BUGREPORT "http://code.google.com/p/yad/issues/entry"
+#define PACKAGE_URL "http://code.google.com/p/yad"
+#define PACKAGE "yad"
+#define VERSION "0.17.1.1"
+
+configure: exit 1
